consensus/src/common.rs

```diff
@@ -117,11 +118,6 @@ pub trait Consensus {
     ) -> Result<&'a ProtocolStateChain, ConsensusError>;
 }
 
-// TODO: replace from checkpoint.rs
-fn is_short_range(_cs: &ProtocolStateChain) -> bool {
```

Comment:
It's conflict with spec https://github.com/MinaProtocol/mina/blob/32f529d8d8d712a44ee75be66061ce08cbdc8924/docs/specs/consensus/README.md#532-short-range-fork-check

Reply:
yes, this has been replaced with the is_short_range function from `checkpoint.rs`

Reply:
Great work on helping modify this part 👍 !